services/minespace-web/src/components/common/DocumentTable.tsx

```diff
@@ -167,6 +171,36 @@ export const DocumentTable = ({
     });
   };
 
+  const handleRowSelectionChange = (value) => {
+    if (documentTypeCode === "") {
+      setDocumentTypeCode(value[0].major_mine_application_document_type_code);
```

Comment:
Not all documents have major_mine_application_document_type_code. I see where we're setting it but not where we're using it- can it be removed? 

I expanded the file looking for references to it and I think there's some unused prop types listed as well, I don't think fileOperationPermissionMap needs to be here, might be more.

Reply:
Removed as discussed.